found non-ASCII code after verticalGridLinesVisible in XYPlot.java

### Subject of the issue
found non-ASCII code after verticalGridLinesVisible in XYPlot.java

### Your environment
* version of jvarkit
* version of java: openjdk-11.0.8_10
* the value of `${JAVA_HOME}`: /opt/spack/opt/spack/linux-centos7-broadwell/intel-19.0.5/openjdk-11.0.8_10-bwcgovtzmcjykzhldbpotpcscyvo6yko
* which OS: Linux 3.10.0-1160.36.2.el7.x86_64 #1 SMP Wed Jul 21 11:57:15 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

### Steps to reproduce
run last step of installation after git clone, cd dir: ./gradlew vcfstatsjfx

### Fix
**It can be fixed** by finding out and removing the non-ASCII code after the variable name "verticalGridLinesVisible". 

### Expected behaviour
Installation success. 

### Actual behaviour
Installation stops after error message:

[ant:javac] warning: [options] bootstrap class path not set in conjunction with -source 8
[ant:javac] /my/folder/software/jvarkit/src/main/java/com/github/lindenb/jvarkit/chart/XYChart.java:120: error: unmappable character (0xE2) for encoding US-ASCII
[ant:javac] public void setVerticalGridLinesVisible(boolean verticalGridLinesVisible???) {
[ant:javac]                                                                         ^
.
.
.
[ant:javac] /my/folder/software/jvarkit/src/main/java/com/github/lindenb/jvarkit/chart/XYChart.java:121: error: not a statement
[ant:javac]     this.verticalGridLinesVisible??? = verticalGridLinesVisible???;
[ant:javac]                                        ^
[ant:javac] /my/folder/software/jvarkit/src/main/java/com/github/lindenb/jvarkit/chart/XYChart.java:124: error: return outside method
[ant:javac]     return verticalGridLinesVisible???;
[ant:javac]     ^
[ant:javac] 18 errors
[ant:javac] 1 warning
> Task :vcfstatsjfx FAILED

FAILURE: Build failed with an exception.

* Where:
Build file '/my/folder/software/jvarkit/build.gradle' line: 676

* What went wrong:
Execution failed for vcfstatsjfx com.github.lindenb.jvarkit.tools.vcfstats.VcfStatsJfx.
> Compile failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s
1 actionable task: 1 executed

--- a/docs/VcfStatsJfx.md
+++ b/docs/VcfStatsJfx.md
@@ -8,6 +8,7 @@ VCF statistics
 ## Usage
 
 ```
+Usage: java -jar dist/vcfstatsjfx.jar  [options] Files
 Usage: vcfstatsjfx [options] Files
   Options:
     --altering, --damaging

--- a/src/main/java/com/github/lindenb/jvarkit/chart/XYChart.java
+++ b/src/main/java/com/github/lindenb/jvarkit/chart/XYChart.java
@@ -75,7 +75,7 @@ public static class Series<X,Y>
 private Axis<X> xAxis;
 private Axis<Y> yAxis;
 private List<XYChart.Series<X,Y>> data;
-private boolean verticalGridLinesVisible​ = false;
+private boolean verticalGridLinesVisible = false;
 
 protected XYChart(final Axis<X> xAxis,final Axis<Y> yAxis, final List<XYChart.Series<X,Y>> data) {
 	this.xAxis = xAxis;
@@ -117,11 +117,11 @@ public void update() {
 		}
 	}
 
-public void setVerticalGridLinesVisible(boolean verticalGridLinesVisible​) {
-	this.verticalGridLinesVisible​ = verticalGridLinesVisible​;
+public void setVerticalGridLinesVisible(boolean verticalGridLinesVisible) {
+	this.verticalGridLinesVisible = verticalGridLinesVisible;
 }
-public boolean isVerticalGridLinesVisible​() {
-	return verticalGridLinesVisible​;
+public boolean isVerticalGridLinesVisible() {
+	return verticalGridLinesVisible;
 }
 
 @Override

